pypeit/pypeit.py

```diff
@@ -481,6 +480,9 @@ def get_std_trace(self, std_redux, det, std_outfile):
             if np.any(this_det):
                 sobjs_det = sobjs[this_det]
                 sobjs_std = sobjs_det.get_std()
+                # No standard extracted??
```

Comment:
Please change this comment to 

#No standard extracted on this detector

Reply:
Done